util/src/dbutil.cpp

```diff
@@ -324,11 +335,16 @@ std::shared_ptr<odb::database> createDatabase(const std::string& connStr_)
       updateConnectionString(connStr_, "database", "postgres");
     std::string dbName = connStrComponent(connStr_, "database");
 
-    createPsqlDatbase(defaultPsqlConnStr, dbName);
-
-    int size = odbOpts.size();
-    db.reset(new odb::pgsql::database(size, cStyleOptions),
-      [](odb::database*){});
+    if(checkPsqlDatbase(defaultPsqlConnStr, dbName, create_))
```

Comment:
Space missing here.

Reply:
Solved in 4b52c300da8e3f8309356cf71e5f9c70207cc051